src/main.cpp

```diff
@@ -28,7 +28,7 @@
 #include <QCommandLineParser>
 #include <QCoreApplication>
 #include <QTimer>
-
+#include <iostream>
```

Comment:
This isn't really necessary, includes should be per file. The main doesn't require `<iostream>`.